site/client/src/main/scala/Ui.scala

```diff
@@ -15,74 +20,86 @@ import model.Exercises._
 import utils.DomHandler._
 
 object UI {
+  val UNSOLVED = "unsolved"
```

Comment:
`case object Unsolved extends ExerciseState("unsolved"). Also there may be a use case for symbols here.